services/pama-imaging.js

```diff
@@ -24,30 +24,60 @@ class Reasons {
     return true;
   }
 
-  constructor(yes, no) {
-    this.yes = yes.map(x => new Set(x));
-    this.no = no.map(x => new Set(x));
+  constructor(appropriate, notAppropriate) {
+    this.appropriate = appropriate.map(x => new Set(x));
+    this.notAppropriate = notAppropriate.map(x => new Set(x));
   }
 
   getRating(reasons) {
-    if (this.yes.filter(s => Reasons.covers(s, reasons)).length) {
+    if (this.appropriate.filter(s => Reasons.covers(s, reasons)).length) {
       return 'appropriate';
     }
-    if (this.no.filter(s => Reasons.covers(s, reasons)).length) {
+    if (this.notAppropriate.filter(s => Reasons.covers(s, reasons)).length) {
       return 'not-appropriate';
     }
     return 'no-guidelines-apply';
   }
 }
 
 const cptReasons = {
-  '1234': new Reasons([['1']], [['2', '3']]), // testing only
+  '1234': new Reasons([['1']], [['2', '3']]),
   '70450': new Reasons([['25064002', '423341008']], []),
   '70544': new Reasons([], []),
+  '71275': new Reasons([], [['13213009']]),
   '72133': new Reasons([], [['279039007']]),
   '75561': new Reasons([['13213009']], []),
 };
 
+const recommendations = {
+  '13213009': [{
```

Comment:
Throughout these tests / hard-coded examples, it _might_ be nice to create a valueset like

```
const SNOMED_CODES = {
 CONGENITAL_HEART_DISEASE: "13213009"
}
```

so that values could be provided like `SNOMED_CODES.CONGENITAL_HEART_DISEASE` for readability. It's hard to tell what's happening at a glance.

Reply:
Good idea, thanks.

Reply:
Done.